README.md

```diff
@@ -1,16 +1,13 @@
 # Metals
 
-<a href="https://travis-ci.org/scalameta/metals">
-<img src="https://img.shields.io/travis/scalameta/metals/main.svg?style=flat-square" />
-</a>
 <a href="https://gitter.im/scalameta/metals">
-<img alt="Join the chat on Gitter" src="https://img.shields.io/gitter/room/scalameta/metals.svg?logo=gitter&style=flat-square&color=F71263" />
```

Comment:
Super nitpicky, but we had different types of icons, and this aligns them to all be the same 😬